### ISSUE TITLE
Make JS literal smarter 

### ISSUE THREAD

**Issue by satyr** [satyr/coco#12]
So that hacks like

```
`EXIT:`
for i from 3 to 5
  for j from 7 to 9
    `break EXIT` if Math.random() < .2
    say i * j
```

works as expected.

**Comment by satyr** [comment]
closed by 7c2e00ffeabbffe9e31fe4cdb2e23b0bd7754b9b; JS literals no longer get semicoloned

**Comment by josher19** [comment]
Oops, this was referencing gkz/LiveScript#12 not coco!
